docker-compose.override.yaml

```diff
@@ -13,3 +11,7 @@ services:
     build: ./app
     volumes:
       - ./app:/home/python/app
+
+  nginx:
+    volumes:
+      - ./nginx/certs:/certs
```

Comment:
Does this mean we use nginx and https for local development, too?

Reply:
Yes. Otherwise the local development would differ quite a bit from the staging/production. It is not only running nginx and https but also the whole port settings would have to be changed by exposing flask again instead of only nginx

Reply:
Okay makes sense!